addon.json

```diff
@@ -0,0 +1,74 @@
+{
+    "identifier": "com.seeq.addons.plot_curve",
+    "name": "Plot Curve",
+    "description": "A tool for fitting curves to tabular data and pushing resulting formulas to Seeq",
+    "version": "1.0.0",
+    "license": "Seeq",
+    "icon": "fa fa-edit",
+    "tags": {
+        "maintainer": "Seeq"
+    },
+    "previews": [],
+    "elements": [
+        {
+            "name": "Plot Curve",
+            "identifier": "com.seeq.addons.plot_curve.datalabfunctions",
+            "known_aliases": ["PlotCurve", "Plot Curve"],
+            "description":"A tool for fitting curves to tabular data and pushing resulting formulas to Seeq",
+            "type": "AddOnTool",
+            "path": "data-lab-functions",
```

Comment:
[nit] It makes no functional difference, but because this is an Add-on Tool and not a Data Lab Functions Add-on, for clarity you might consider replacing `data-lab-functions` everywhere in this PR with `add-on-tool`.

Reply:
In the next push!